### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by Natali-Pchelkina**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="./main.js"></script>
```
Comment (ghost) [inline index.html]: Should be added before closing `body`.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+let test = 1;
+++test;
+test += 1;
+console.log(test); //В консоле значение 3//
+test -= 1;
+console.log(test); //В консоле значение 2//
+console.log(Boolean(test)); //В консоле значение 'true'//
+
+//Task 9//
+let arr = [2, 3, 5, 8];
```
Comment (ghost) [inline main.js]: Should be a `const`.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+let result = 1;
+for(let i = 0; i<arr.length; i++) {
+    result *= arr[i];
+};
+console.log('result', result);
+
+//Task 10
+let arr10 = [2, 5, 8, 15, 0, 6, 20, 3];
```
Comment (ghost) [inline main.js]: Should be a `const`.

File: main.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+for(let i = 0; i < arr10.length; ++i) {
+    if(arr10[i] > 5 && arr10[i] < 10) {
+    console.log(arr10[i]);
+    }
+}
+
+//Task 11
+let arr11 = [2, 5, 8, 15, 0, 6, 20, 3];
```
Comment (ghost) [inline main.js]: Should be a `const`.

File: main.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+for(let i = 0; i < arr11.length; ++i) {
+    if(!(arr11[i] % 2)) {
+        console.log(arr11[i]);
+    }
+}
+
+ 
+
```
Comment (ghost) [inline main.js]: plz, delete empty lines

File: index.html (hunk 1 above)
Comment (Natali-Pchelkina) [reply index.html]: <script> added before closing body

File: main.js (hunk 2 above)
Comment (Natali-Pchelkina) [reply main.js]: Replaced the variable

File: main.js (hunk 3 above)
Comment (Natali-Pchelkina) [reply main.js]: Replaced the variable

File: main.js (hunk 4 above)
Comment (Natali-Pchelkina) [reply main.js]: Replaced the variable

File: main.js (hunk 5 above)
Comment (Natali-Pchelkina) [reply main.js]: deleted empty lines

**Pull request merged**